Dockerfile

```diff
@@ -24,8 +24,12 @@ RUN pip3 install --upgrade -r requirements.txt
 RUN curl -L https://mirror.openshift.com/pub/openshift-v4/clients/ocp/4.1.7/openshift-client-linux-4.1.7.tar.gz | tar xvzf -
 COPY *.py oc_in_cluster.sh /
 RUN chmod a+r /*.py && \
-    chmod a+rx /oc_in_cluster.sh
-ENTRYPOINT ["python3", "/workload_runner.py"]
+    chmod a+rx /oc_in_cluster.sh && \
+    chmod a+rx /chaos_runner.py
+
+RUN bash oc_in_cluster.sh
```

Comment:
What does this do? `oc_in_cluster.sh` is designed to be used from the pod to run "oc" commands.